src/main/java/http/logger/Logger.java

```diff
@@ -0,0 +1,22 @@
+package http.logger;
+
+import http.request.HttpRequest;
```

Comment:
This class shouldn't have a hard dependency on HttpRequest. It makes it so it is not reusable without modification.

Reply:
👍 noted and changed